### PR TITLE
Add already exist error

### PR DESCRIPTION
**Pull request by tomahawk28**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/giantswarm/appcatalog v0.4.1
 	github.com/giantswarm/backoff v0.2.0
 	github.com/giantswarm/exporterkit v0.2.1
-	github.com/giantswarm/helmclient/v4 v4.5.0
+	github.com/giantswarm/helmclient/v4 v4.5.1-0.20210430080600-653555481d8d
```
Comment (tomahawk28) [inline go.mod:11]: Move it to v4.6.0 after merge & release new version from https://github.com/giantswarm/helmclient/pull/341

**Comment by tomahawk28** [comment]
Tested in `geckon` 
```
 kg get charts releases-kvm-unique                                                                                                                                      ⎈ (giantswarm-geckon/default)NAME                  VERSION                                          LAST DEPLOYED   STATUS
releases-kvm-unique   0.0.0-c43aadfdb012113167bca2aaa28f7bb789b77385                   already-exists
```

File: service/controller/chart/resource/release/create.go [hunk 2]
```diff
@@ -112,7 +112,16 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		r.logger.Debugf(ctx, "canceling resource")
 		return nil
 	}
-	if helmclient.IsValidationFailedError(err) {
+	if helmclient.IsResourceAlreadyExists(err) {
```
Comment (tomahawk28) [inline service/controller/chart/resource/release/create.go:115]: Even though the original issue was from update events, we also need to care for create.go as well.

**Review by rossf7 — APPROVED** [review]
LGTM

**Pull request merged**